server/game/cards/06.6-TBWB/KingsBlood.js

```diff
@@ -0,0 +1,46 @@
+const _ = require('underscore');
+
+const DrawCard = require('../../drawcard.js');
+
+class KingsBlood extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Discard power from opponent\'s faction',
+            condition: () => {
+                //Possible TODO: incorporate snapshot functionality
+                if(this.hasToken('gold')) {
+                    this.parentCard = this.parent;
+                    this.gold = this.tokens['gold'];
+                    return true;
+                }
+                return false;
+            },
+            phase: 'plot',
+            cost: [
+                ability.costs.kneelParent(),
+                ability.costs.sacrificeSelf()
+            ],
+            handler: context => {
+                _.each(this.game.getPlayers(), player => {
```

Comment:
While identical in functionality, looping through `this.game.getOpponents(context.player)` would closer match the card text and thus read better (and also not need the extra if)